content/md/imports-exports/automate-imports-and-exports.md

```diff
@@ -0,0 +1,49 @@
+---
+id: import-export-automation
+themes: import-export-data
+title: Automate and schedule all imports/exports
+popular: false
+ee: true
+ge: true
+related: imports, exports, tailored-export, tailored-import,
+---
+
+# Overview
+
+This feature helps you to automatize your product imports and/or exports, and is available only for our Enterprise Edition or Growth Edition users.
+You will be able to:
+* Connect your product imports/exports to remote SFTP storages.
+* Automate them by scheduling their executions.
```

Comment:
Same as above, I don't know it is a good idea to speak about not yet released feature in the doc